src/Security/SecIdentity2.cs

```diff
@@ -14,13 +14,20 @@
 using System;
 using System.Runtime.InteropServices;
 using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
 using ObjCRuntime;
 using Foundation;
 using CoreFoundation;
 
 namespace Security {
 
-	[TV (12,0), Mac (10,14), iOS (12,0), Watch (5,0)]
+#if !NET
+	[iOS (12,0), TV (12,0), Mac (10,14), Watch (5,0)]
```

Comment:
AS with the other PRs and as requested by @spouliot  lets not change the original.